singlestoredb/apps/_config.py

```diff
@@ -64,3 +64,46 @@ def token(self) -> Optional[str]:
             return self.app_token
         else:
             return self.user_token
+
+
+@dataclass
+class PythonUdfAppConfig:
```

Comment:
I think naming this `UDFAppConfig` would be better. Since there is no other option than Python. `Udf` is a bit too Java-esque. I prefer `UDF` .